adminpages/wizard/done.php

```diff
@@ -0,0 +1,90 @@
+<?php
+	$site_type = get_option( 'pmpro_site_type', true );
+
+	if ( empty( $site_type ) ) {
+		$site_type = __( 'general', 'paid-memberships-pro' );
+	}
+	// Get Add On recommendations based on site type.
+	$addon_cats = pmpro_get_addon_categories();
+	
+	if ( ! empty( $addon_cats[$site_type] ) && $addon_cats[$site_type] ) {
+		$addon_list = $addon_cats[$site_type];
+	} else {
+		$addon_list = $addon_cats['popular'];
+	}
+
+	// Did they choose collect payments? If so, show a nudge to complete the gateway setup.
+	$configure_payment = pmpro_getOption( 'wizard_collect_payment', true );
+
+?>
+<div class="pmpro-wizard__step pmpro-wizard__step-4">
+				<div class="pmpro-wizard__step-header">
+					<h1><?php esc_html_e( 'Setup Complete', 'paid-memberships-pro' ); ?></h1>
+					<p><strong>Congratulations!</strong> Your membership site is ready.</p>
+				</div>
+				<div class="pmpro-wizard__field"> <!-- Recommended icons -->
+					<h1>What's next?</h1>
+					<p><?php _e( sprintf( "You indicated you're building a %s membership site. Here are some recommended Add Ons for your business.", "<strong>$site_type</strong>" ), 'paid-memberships-pro' ); ?></p>
```

Comment:
`$site_type` can be an underscored string like `digital_downloads` or `physical_products`. We should format it better. This may just involve converting underscores to spaces. This string also needs to be escaped.